### DISCUSSION TITLE
Limiting CAYW to specific collection

### DISCUSSION THREAD

**Discussion by Dave-Novo** [retorquere/zotero-better-bibtex#2731] [Q&A] [answered]
Hello. I am just starting with better bibtex for zotero.
I have a collection within zotero that I have exported to a .bib file with the "keep updated"  flag set and that works fine. 

I am using TexStudio, and have created the following macro

```
var url = "http://127.0.0.1:23119/better-bibtex/cayw??format=biblatex&command=autocite&texstudio=true&minimize=true"
system("curl -s " + url)
```
There are two problems:
1. the CAYW picker shows all the files in my library, not sure the collection I exported. For this latex file, I just want to limit it to references in the collection (hence I only exported the collection). Is there a way to pass the name of the collection I want the picker to search through as an argument to cayw? Looking in the docs, it does not appear that is the case. Am I missing something, or would this be a feature request?

**Comment by retorquere** [comment]
That would be a feature request.

**Reply by retorquere** [reply, marked as answer]
Ah wait, you mean the picker itself. That's out of my control unfortunately, I just ask zotero to put up its picker and get its result. I can limit the results, but not what shows up in the picker.

**Comment by Dave-Novo** [comment]
That is a shame. Does Zotero's picker not have options that you can pass through to it to control what it shows?

**Reply by retorquere** [reply]
No, it's just not how zotero works. There's an option to limit the picker to a single library, but to the extent of my knowledge, that's it.
